src/app/workflow/executions/executions-tab.component.ts

```diff
@@ -0,0 +1,218 @@
+/*
+ *    Copyright 2023 OICR, UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+import { Component, Input, OnChanges } from '@angular/core';
+import { EntryTab } from '../../shared/entry/entry-tab';
+import {
+  CloudInstance,
+  CpuMetric,
+  ExecutionTimeMetric,
+  ExtendedGA4GHService,
+  MemoryMetric,
+  Metrics,
+  WorkflowVersion,
+} from '../../shared/openapi';
+import { WorkflowQuery } from '../../shared/state/workflow.query';
+import { SessionQuery } from '../../shared/session/session.query';
+import { takeUntil } from 'rxjs/operators';
+import { BioWorkflow, Notebook, Service } from '../../shared/swagger';
+import { CheckerWorkflowService } from '../../shared/state/checker-workflow.service';
+import { CheckerWorkflowQuery } from '../../shared/state/checker-workflow.query';
+import PartnerEnum = CloudInstance.PartnerEnum;
+import { MatSelectChange } from '@angular/material/select';
+import { AlertService } from '../../shared/alert/state/alert.service';
+
+interface MetricsTableObject {
+  metric: string;
+  min: number;
```

Comment:
On line 157 you have `min: metricsData?.minimum`, which means the value of `min` can be undefined or null, so you should make this `min?: number`. Same comment for any other applicable fields in this interface and potentially ValidationsTableObject that can be null/undefined.